specifications.yml

```diff
@@ -68,104 +80,155 @@ paths:
         500:
           $ref: "#/components/responses/serverError"
       security:
-      - oAuth2ClientCredentials: []
+        - oAuth2ClientCredentials: []
 
-  /application/order:
-    post:
+  # /financing-application/{applicationId}: # Do not release for now
+  #   get:
+  #     tags:
+  #     - Factoring endpoints
+  #     summary: Get a financing application
+  #     description: Fetch a financing application
+  #     parameters:
+  #     - name: applicationId
+  #       in: path
+  #       description: Application's id
+  #       required: true
+  #       schema:
+  #         type: string
+  #     responses:
+  #       200:
+  #         $ref: "#/components/responses/financingApplicationGetSuccess"
+  #       400:
+  #         $ref: "#/components/responses/dataError"
+  #       401:
+  #         $ref: "#/components/responses/authError"
+  #       404:
+  #         $ref: "#/components/responses/notFoundError"
+  #       500:
+  #         $ref: "#/components/responses/serverError"
+  #     security:
+  #       - oAuth2ClientCredentials: []
+
+  # /customer-application/create: # We need to update request body and response
+  #   post:
+  #     tags:
+  #     - Application endpoints
+  #     summary: Create a customer
+  #     description: Create a customer and return its credit information. If a customer with the defined orgNumber has already been added to the system, it returns the existing one.
+  #     requestBody:
+  #       $ref: "#/components/requestBodies/createCustomerApplication"
+  #     responses:
+  #       200:
+  #         $ref: "#/components/responses/customerGetSuccess"
+  #       400:
+  #         $ref: "#/components/responses/dataError"
+  #       401:
+  #         $ref: "#/components/responses/authError"
+  #       500:
+  #         $ref: "#/components/responses/serverError"
+  #     security:
+  #       - oAuth2ClientCredentials: []
+
+  /customer-application/{applicationId}: # We need to update response 
```

Comment:
I think this is fine if this works. How much extra wrk is it to add the response as well?

Reply:
I think around 1-2 hours, the work left to do here is to write the schema for customer application toJson